### PR TITLE
NotroDev.SkEditor+ version 1.7.8

### PR DESCRIPTION
**Pull request by CuriousDog81**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/118152)

### CODE REVIEW

**Comment by CuriousDog81** [comment]
@microsoft-github-policy-service agree

File: manifests/n/NotroDev/SkEditor+/1.7.8/NotroDev.SkEditor+.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: NotroDev.SkEditor+
+PackageVersion: 1.7.8
+InstallerType: msi
+ProductCode: '{29E3F9DC-A3F3-499B-A230-3C9F02903C82}'
```
Comment (stephengillie) [inline manifests/n/NotroDev/SkEditor+/1.7.8/NotroDev.SkEditor+.installer.yaml:7]:
```suggestion
ProductCode: '{29E3F9DC-A3F3-499B-A230-3C9F02903C82}'
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.6
```

Comment (stephengillie) [reply manifests/n/NotroDev/SkEditor+/1.7.8/NotroDev.SkEditor+.installer.yaml:7]: image

Comment (CuriousDog81) [reply manifests/n/NotroDev/SkEditor+/1.7.8/NotroDev.SkEditor+.installer.yaml:7]: I forgot to try on a fresh vm without it, sorry

**Pull request closed without merging**